src/app/ApplicationTemplate.Tests/Presentation/NavigationShould.cs

```diff
@@ -1,45 +1,67 @@
-﻿using System.Threading.Tasks;
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
 using ApplicationTemplate.Presentation;
+using Chinook.SectionsNavigation;
 using Xunit;
 
 namespace ApplicationTemplate.Tests
 {
 	public partial class NavigationShould : NavigationTestsBase
 	{
 		[Fact]
-		public async Task NavigateEverywhere()
+		public async Task MenuNavigationShould()
```

Comment:
I though test classes were supposed to have the "Should" suffix, not the actual test methods.
It doesn't seem to work right now:
"NavigationShould.MenuNavigationShould" doesn't make much sense.

Reply:
"MenuNavigationShould" doesn't tell us anything about what should happen.